README.md

```diff
@@ -135,6 +135,34 @@ val yaml = Color("CadetBlue", 95, 158, 160).toYaml
 val color = yaml.convertTo[Color]
 ```
 
+Example with inner objects:
+
+```scala
+case class Color(name: String, red: Int, green: Int, blue: Int)
+case class Pallete(name: String, colors: Option[List[Color]] = None)
+
+object PalletleYamlProtocol extends DefaultYamlProtocol {
```

Comment:
The `PalletleYamlProtocol` is a typo. However, for this example, I don't think it is necessary for you to have the `YamlFormats` inside an object. It would simplify the example and avoid the import in [L149](https://github.com/jcazevedo/moultingyaml/pull/13/commits/f76f693e06a2c11c5361e9f27e07686fe248403d#diff-04c6e90faac2675aa89e2176d2eec7d8R149).